main/src/library/Console.flix

```diff
@@ -733,7 +761,7 @@ namespace Console {
         else if (isAnsiTerminal())
             ColorSupport.Ansi
         else
-            ColorSupport.None
+            ColorSupport.NoColor
 
```

Comment:
So I guess nothing is done for StdErr because it is not supported by java's console?

Reply:
I was a bit in doubt of what to do with the namespaces. StdOut and StdIn works like they should. They can write to their streams. StdIn doesn't without global state seemingly so I don't know how to fix that. So this PR is more like adding access to the java console class, not really fixing StdIn.